### ISSUE TITLE
AntD button not rendering it's default color

### ISSUE THREAD

**Issue by maifeeulasad** [ant-design/ant-design#41089]
### Reproduction link

[https://github.com/maifeeulasad/remanage/tree/3b64145a2b75707559cae8f5144993f9288fb927](https://github.com/maifeeulasad/remanage/tree/3b64145a2b75707559cae8f5144993f9288fb927)

### Steps to reproduce
```
yarn && yarn start
```
### What is expected?

These two buttons at the top should have their default color right? But it's not rendering it's default color. Although the border is rendering fine.

### What is actually happening?

It's rendering transparent I guess. But on hover it's rendering the default sky blue color.

| Environment | Info |
| --- | --- |
| antd | 4.22.3 |
| React | 18.1.0 |
| System |  ubuntu-latest |
| Browser | Chrome / Edge |

Here is a snap:

![image](https://user-images.githubusercontent.com/29339330/223130523-699315f6-b419-40c5-9b18-c8c4d568f689.png)

### Additional info

It was first intercepted after I entroduced *manual code splitting*, here:

https://github.com/maifeeulasad/remanage/blob/3b64145a2b75707559cae8f5144993f9288fb927/vite.config.ts#L23-L27

**Comment by zombieJ** [comment]
![223130523-699315f6-b419-40c5-9b18-c8c4d568f689](https://user-images.githubusercontent.com/5378891/223132465-92452491-e697-4d15-a492-4d56d7998cb0.png)

Check your code, it seems override by other style which not belong to antd.

**Comment by maifeeulasad** [comment]
Dear @zombieJ,

Turns out there were no css importing issues in my code. But as I intruduced `rollupOptions.output.manualChunks` in my code recently, it was splitting the `.css` file along with all the `.js` files. So it was creating some issue, I'm still not sure, what issue was that.

But I resolved it simply by excluding all the `.css` file from splitting.

Maybe it will help someone:
![exclude-css-from-manualChunks-to-render-antd-properly-with-vite](https://user-images.githubusercontent.com/29339330/223639250-15bceecd-9c80-4ca7-a3b4-2f5b1ea5d918.png)

Ref: https://github.com/maifeeulasad/remanage/commit/1b81b81c38dc0f4dc8c0f165cd8dfbc335e83c1b

But, it still leaves us with one problem, and that is bulky css. I guess, we can study more, and fix it.

Good day.🍷
